appengine-java8/tasks/src/main/java/com/example/task/CreateHttpTask.java

```diff
@@ -0,0 +1,77 @@
+/*
+ * Copyright 2018 Google LLC
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package com.example.task;
+
+import com.google.cloud.tasks.v2beta3.CloudTasksClient;
+import com.google.cloud.tasks.v2beta3.HttpMethod;
+import com.google.cloud.tasks.v2beta3.HttpRequest;
+import com.google.cloud.tasks.v2beta3.QueueName;
+import com.google.cloud.tasks.v2beta3.Task;
+import com.google.common.base.Strings;
+import com.google.protobuf.ByteString;
+import com.google.protobuf.Timestamp;
+
+import java.nio.charset.Charset;
+import java.time.Clock;
+import java.time.Instant;
+
+import org.apache.commons.cli.CommandLine;
+import org.apache.commons.cli.CommandLineParser;
+import org.apache.commons.cli.DefaultParser;
+import org.apache.commons.cli.HelpFormatter;
+import org.apache.commons.cli.Option;
+import org.apache.commons.cli.Options;
+import org.apache.commons.cli.ParseException;
+
+public class CreateHttpTask {
+
+  public static void main(String[] args) throws Exception {
+    String projectId = System.getenv("PROJECT_ID");
+    String queueName = System.getenv("QUEUE_ID");
+    String location = System.getenv("LOCATION_ID");
+    String url = System.getenv("URL");
+
+    // [START cloud_tasks_create_http_task]
```

Comment:
Please put the START tag above import statements so that the dependency paths are included when the code is hosted on c.g.c. (and move the END tag past close brackets as necessary).

Reply:
done.